src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,24 @@
+"""Kubectl Client BCI container"""
+
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package.versions import get_pkg_version
+
+KUBECTL_CONTAINERS = []
+KUBECTL_SUPPORTED_OS_VERSIONS = [
+    OsVersion.TUMBLEWEED,
+]
+
+for os_version in KUBECTL_SUPPORTED_OS_VERSIONS:
+    KUBECTL_CONTAINERS.append(
+        ApplicationStackContainer(
+            name="kubectl",
+            os_version=os_version,
+            version=get_pkg_version("kubernetes", os_version),
+            license="MPL-2.0",
+            is_latest=True,
+            pretty_name="Kubectl Client",
```

Comment:
there is  a multistage missing, e.g. from_target_image.

Reply:
This is not strictly necessary, building on base is fine

Reply:
multistage is not added for now as it is not necessary.